flake.nix

```diff
@@ -0,0 +1,58 @@
+{
+  description = "Codex build flake";
+
+  inputs = {
+    nixpkgs.url = "github:NixOS/nixpkgs/nixos-unstable";
+    flake-utils.url = "github:numtide/flake-utils";
+  };
+
+  outputs = { self, nixpkgs, flake-utils }:
+    flake-utils.lib.eachDefaultSystem (system:
```

Comment:
I think I prefer avoiding `flake-utils`, the same can be achieved with a few lines:
https://github.com/status-im/go-waku/blob/3c4e40c729a0ad5f1079ac1e3f31eeaae1b9cbdd/flake.nix#L8-L13

But I'm not saying you have to change it.